hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocol/OzoneManagerProtocol.java

```diff
@@ -983,4 +983,16 @@ EchoRPCResponse echoRPCReq(byte[] payloadReq,
                              int payloadSizeResp)
           throws IOException;
 
+
+  /**
+   * Start the lease recovery of a file.
+   *
+   * @param volumeName - The volume name.
+   * @param bucketName - The bucket name.
+   * @param keyName - The key user want to recover.
+   * @return true if the file is already closed
+   * @throws IOException if an error occurs
+   */
+  boolean recoverLease(String volumeName, String bucketName,
+                              String keyName) throws IOException;
```

Comment:
The client related methods (this and many other methods) should be moved to `OzoneManagerClientProtocol`.  We could do it in a separated JIRA.

Reply:
Thanks for the information. I wasn't aware of it, simply following the past implementations.
OzoneManagerClientProtocol looks short. Are we supposed to move all client to OM communications over to this interface, and what about the existing APIs?

Reply:
Let's keep it as-is and do it separately.  It will be an easy but big change.